src/operator/src/req_convert/insert/stmt_to_region.rs

```diff
@@ -134,7 +135,18 @@ impl<'a> StatementToRegion<'a> {
         let requests = Partitioner::new(self.partition_manager)
             .partition_insert_requests(table_info.table_id(), Rows { schema, rows })
             .await?;
-        Ok(RegionInsertRequests { requests })
+        let requests = RegionInsertRequests { requests };
+        Ok(if table_info.is_ttl_instant_table() {
+            InstantInsertRequests {
+                normal_requests: Default::default(),
+                instant_requests: requests,
+            }
+        } else {
+            InstantInsertRequests {
+                normal_requests: requests,
+                instant_requests: Default::default(),
+            }
+        })
```

Comment:
non-blocking: Use the same style.
```suggestion
        if table_info.is_ttl_instant_table() {
            Ok(InstantInsertRequests {
                normal_requests: Default::default(),
                instant_requests: requests,
            })
        } else {
            Ok(InstantInsertRequests {
                normal_requests: requests,
                instant_requests: Default::default(),
            })
        }
```